magefiles/magefile.go

```diff
@@ -93,6 +92,11 @@ func Vet() {
 	must.RunV("go", "vet", "./...")
 }
 
+func Lint() {
+	tools.EnsureStaticCheck()
```

Comment:
I can't remember if we do it like this elsewhere. Normally, we would call EnsureStaticCheck with mg.Dep so that it's only called once even when multiple targets call Lint. Is there a reason why we aren't doing that here?

e.g.

```suggestion
	mg.Dep(tools.EnsureStaticCheck)
```

Reply:
I fixed in porter as well: https://github.com/getporter/porter/pull/2515/files